mods/bones/init.lua

```diff
@@ -186,18 +186,39 @@ minetest.register_on_dieplayer(function(player)
 		bones_mode = "bones"
 	end
 
+	local enable_bones_logging = minetest.settings:get_bool("enable_bones_logging")
```

Comment:
`minetest.settings:get_bool("enable_bones_logging") ~= false` if you want to default this setting to `true`

Reply:
Little addition: I commented on this line because you're specifying the `enable_bones_logging` default value to `true in the settingtypes.txt file but that's (yet) only used for the Advanced Settings GUI in the main menu, so it doesn't have an effect on the actual defaults.